engine/testabilities/fixture_engine.go

```diff
@@ -38,6 +38,9 @@ type EngineFixture interface {
 
 	// BHS creates a new test fixture for Block Header Service (BHS)
 	BHS() BlockHeadersServiceFixture
+
+	// ARC creates a new test fixture for ARC
+	ARC() ARCFixture
```

Comment:
maybe it's good idea to setup some default for ARC? Like, return success by default, or return error.

Reply:
We're leaving it for later.